doc/source/user_guide/sparse.rst

```diff
@@ -71,30 +115,11 @@ attributes and methods that are specific to sparse data.
 This accessor is available only on data with ``SparseDtype``, and on the :class:`Series`
 class itself for creating a Series with sparse data from a scipy COO matrix with.
 
-.. _sparse.array:
-
-SparseArray
------------
 
-``SparseArray`` is the base layer for all of the sparse indexed data
-structures. It is a 1-dimensional ndarray-like object storing only values
-distinct from the ``fill_value``:
-
-.. ipython:: python
-
-   arr = np.random.randn(10)
-   arr[2:5] = np.nan
-   arr[7:8] = np.nan
-   sparr = pd.SparseArray(arr)
-   sparr
-
-Like the indexed objects (SparseSeries, SparseDataFrame), a ``SparseArray``
-can be converted back to a regular ndarray by calling ``to_dense``:
-
-.. ipython:: python
-
-   sparr.to_dense()
+.. versionadded:: 0.25.0
 
+A ``.sparse`` accessor has been added for :class:`DataFrame` as well.
+See :ref:`api.dataframe.sparse` for more.
 
 SparseIndex objects
```

Comment:
I think we should basically drop the docs on SparseIndex, these are an internal representation.